amosbank.cc

```diff
@@ -4,6 +4,7 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdint.h>
```

Comment:
:+1: This is the right thing to do. I wonder through which header it gets included on the systems I tested on...